client/my-sites/stats/activity-log/index.jsx

```diff
@@ -100,6 +100,13 @@ class ActivityLog extends Component {
 		window.scrollTo( 0, 0 );
 	}
 
+	handlePeriodChange = ( { date, direction } ) => {
+		this.props.recordTracksEvent( 'calypso_activitylog_monthpicker_change', {
```

Comment:
A bit I find confusing in this event is that for the reader it's not 100% clear whether this date is the date before the change event occurred, or the date after the change event occurred. Do you think we can make it more clear in any way? Perhaps rename the `date` property to something more self-explicable, like `dateAfterChange` or something?